_posts/2018-11-16-JOIN-2018.md

```diff
@@ -0,0 +1,533 @@
+
+## Deep learning in practice by Remco Runge
+<span class="image left"><img class="p-image" alt="Remco Runge" src="/img/join-2018/remco-speaker-zoom.jpg"></span>
+
+Machine learning: you try to get the good algorithm to find the stuff you want to find.
+
+Deep Learning is based on biology, about how people thought the brain functioned. 
+The assumption was proven wrong but the basic principles still work.
+
+Tensorflow has a nice [playground](https://playground.tensorflow.org/){:target="_blank"} where you can see these effects live in action.
+
+Deep learning has two flows:
+* Forward propagation: feed data through it.
+* Backward propagation: update weights so the next iteration will be better.
+
+You can define weights on input in order to make certain inputs more important.
+
+Deep learning requires a very large network with a large amount of nodes and a lot of hidden layers.
+
+The early layers allow your network to distinguish basic structures within your data.
+Further down in the layers the network will learn more detailed features like; eyes, ears, ... 
+
+It is cool that the network learns these features about how to recoginize a person just by providing it with pictures of faces. 
+We did not provide it with the concept `eyes` but eventually, in one of the iterations the network started to take eyes into consideration.
+
+### Deep Learning at Ordina
+
+#### Workshop: hack a drone.
+A small drone was equiped with deep learning to detect and identify objects by using the [Deep Learning for Java](https://deeplearning4j.org/){:target="_blank"}.
+In a short demo we saw it could identify a notebook.
+
+#### Tic Tac Toe
+In contrary to [Go](https://en.wikipedia.org/wiki/Go_(game)){:target="_blank"}, Tic Tac Toe just has 360.000 possible moves making it easier to use for a demo.
+
+The network for this demo learned by playing against some random opponents without any real strategy, this made it quite dumb. 
+You could improve this by letting it play against itself.
+
+#### Digital Railway Survey
+The idea is to recognize signs next to the railway and verify if these have been installed at the correct place.
+
+[You Only Look Once](https://en.wikipedia.org/wiki/Object_detection){:target="_blank"} is a very cool algorithm which gives you very nice bounded boxes around the objects on the images making it easier for your network to know where it needs to look and thus preventing you from processing unrelevant pictures.
+
+It was also nice to see that with deep learning the system can distinguish signs from a passing train even when we would be unable to see these signs, let alone see what has been written on it.
+
+### Questions
+#### How long does it take to get started?
+If you work with existing data sets and known algorithms, 15 lines of code is enough.
+
+Remco mentioned that for a brand new project it can easily take up 55 days just to show that deep learning is possible for a specific business case.
```

Comment:
```suggestion
Remco mentioned that for a brand new project it can easily take up to 55 days just to show that deep learning is possible for a specific business case.
```